test/post-911-gib-status/01-authed.e2e.spec.js

```diff
@@ -0,0 +1,48 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts.js');
+const GibsHelpers = require('../e2e/post-911-gib-status-helpers.js');
+const LoginHelpers = require('../e2e/login-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const token = LoginHelpers.getUserToken();
+
+    GibsHelpers.initApplicationMock(token);
+
+    // Ensure main status page renders.
+    LoginHelpers.logIn(token, client, '/education/gi-bill/post-9-11/ch-33-benefit', 3)
+      .waitForElementVisible('body', Timeouts.normal)
+      .axeCheck('.main')
+      .assert.title('Check Benefit: Vets.gov')
+      .waitForElementVisible('.schemaform-title', Timeouts.slow);  // First render of React may be slow.
+
+    // Checking field in UserInfoSection has rendered
+    client.expect.element('#fullName').text.to.contain('First Last');
+
+    // Checking field in EnrollmentHistory has rendered
+    client.expect.element('#enrollment-0 h4').text.to.contain('11/01/2012 to 12/01/2012 at Purdue University');
+
+    // Click on print page
+    client
+      .click('#print-button')
+      .windowHandles((result) => {
+        const newWindow = result.value[1];
+        client.switchWindow(newWindow);
+      });
+
+    // Ensure print page renders.
+    // We need to log in again here because Electron loses the session
+    // when opening a new window.
+    LoginHelpers.logIn(token, client, '/education/gi-bill/post-9-11/ch-33-benefit/print', 3)
+      .waitForElementVisible('body', Timeouts.normal)
+      .axeCheck('.main')
+      .assert.title('Check Benefit: Vets.gov')
```

Comment:
Tried both of these suggestions and neither has passed Jenkins :(. What has most recently been pushed is moving the entire block into `.window_handles` (I also changed it from `windowHandles` to `window_handles` because that's what the nightwatch docs specify: http://nightwatchjs.org/api/switchWindow.html).

Everything is still passing locally, just not in Jenkins. Might ping @robbiethegeek to see if he can take a look at this and see if anything else might be going on.